tests/functional/includes/html.event.polyfill.js

```diff
@@ -0,0 +1,9 @@
+// Needed for PhantomJS < 2.0
+
+if (!window.Event || typeof Event !== 'function') {
```

Comment:
I'm only aware of the one in the FT service https://github.com/Financial-Times/polyfill-service/blob/master/polyfills/Event/polyfill.js